server-client/index.js

```diff
@@ -204,6 +204,12 @@ export class ServerClient {
       return true
     }
 
+    if (this.app.queueTypes.includes(type) && !queueCall) {
+      let queue = this.app.queues.get(this.key)
```

Comment:
Note that we should have separated queue for each queue and client https://github.com/logux/logux/issues/54

> For every client and queue, we are creating action queue. 

Here you put all client to the same queue

Reply:
When a new client is connected, a new queue is declared for it and put in a map in `BaseServer.addClient`. So, the queue in this function should be a particular queue, belonging to this client.